### ISSUE TITLE
issue(sourcemaps): sourcemaps not being generated 

### ISSUE THREAD

**Issue by dylanvdmerwe** [ionic-team/ionic-app-scripts#1297]
#### Short description of the problem:
`ionic cordova run android` does not seem to generate sourcemaps anymore. When connecting to the app over `chrome://inspect` you cannot see the ts files and debug nicely like you used to. I haven't done Android dev in a while so not sure when this changed

**Which @ionic/app-scripts version are you using?**
3.0.1

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)
cli packages: (/Users/dylan/Desktop/Projects/CochranePlus/node_modules)

    @ionic/cli-utils  : 1.15.0
    ionic (Ionic CLI) : 3.15.0

global packages:

    cordova (Cordova CLI) : 7.1.0

local packages:

    @ionic/app-scripts : 3.0.1
    Cordova Platforms  : ios 4.5.2
    Ionic Framework    : ionic-angular 3.8.0

System:

    ios-deploy : 1.9.2
    ios-sim    : 6.1.2
    Node       : v6.11.4
    npm        : 4.6.1
    OS         : macOS Sierra
    Xcode      : Xcode 9.0.1 Build version 9A1004

Environment Variables:

    ANDROID_HOME : not set

Misc:

    backend : legacy

**Comment by AlQuraian** [comment]
Have you tried setting a break point in your JS file? When you do this Chrome will navigate you away to the TS file.

**Comment by dylanvdmerwe** [comment]
No, this should work as per `ionic serve` in browser.

**Comment by AlQuraian** [comment]
Yes, that's what I mean. In the browser, when you set a breakpoint it will open the TS file that has the code you just set a break point on.
Then you can debug from the TS file.

**Comment by yurik94** [comment]
Sane bere, no TS files...
